src/main/java/fi/otavanopisto/kuntaapi/server/rest/OrganizationsApiImpl.java

```diff
@@ -578,7 +577,8 @@ public Response getOrganizationTileImageData(String organizationIdParam, String
 
   @Override
   public Response createOrganizationSetting(String organizationIdParam, OrganizationSetting setting) {
-    OrganizationId organizationId = toOrganizationId(organizationIdParam);
+    return createNotImplemented(NOT_IMPLEMENTED);
+    /*OrganizationId organizationId = toOrganizationId(organizationIdParam);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)